src/software/ai/hl/stp/stp.cpp

```diff
@@ -324,12 +335,5 @@ std::map<std::shared_ptr<const Tactic>, Robot> STP::assignRobotsToTactics(
         robots = remaining_robots;
     }
 
-    // store readable assignment map for PlayInfo
-    readable_robot_tactic_assignment.clear();
-    for (const auto& [tactic, robot] : robot_tactic_assignment)
-    {
-        readable_robot_tactic_assignment.emplace(robot.id(), objectTypeName(*tactic));
-    }
```

Comment:
was this not being used anywhere?

Reply:
`robot_tactic_assignment` is a member variable now, so we don't need to store `readable_robot_tactic_assignment` anymore